query-engine/core/src/interactive_transactions/actor_manager.rs

```diff
@@ -63,7 +63,15 @@ impl TransactionActorManager {
             self.send_done.clone(),
         );
 
-        self.clients.write().await.insert(tx_id, client);
+        self.clients.write().await.insert(tx_id.clone(), client);
```

Comment:
Is this new clone required?

Reply:
Ah good spot. That was a legacy from my debugging.